### PR TITLE
added #1 OpenStreetMap/Nominatim provider

### PR DESCRIPTION
**Pull request by kmonmousseau**

### CODE REVIEW

File: Resources/doc/provider/nominatim.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+Nominatim API
+=============
+
+Doc
+---
+
+Go to [www.bingmapsportal.com](http://wiki.openstreetmap.org/wiki/Nominatim).
```
Comment (methylbro) [inline Resources/doc/provider/nominatim.md]: change the link ancor please.

File: Tests/Provider/Nominatim/LocatorTest.php [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+            ->setLongitude(3.8726549)
+        ;
+
+        $address = $this
+            ->getLocator('get-address-result.json')
+            ->locate($coordinates)
+        ;
+
+        $this->assertEquals(
+            'Rue du Thor, Millénaire, Port Marianne, Montpellier, Hérault, Languedoc-Roussillon, France métropolitaine, 34000;34070;34080;34090, France',
```
Comment (methylbro) [inline Tests/Provider/Nominatim/LocatorTest.php]: Maybe you should build a more lisible result.
With a concatenation of `address.road`, `address.city`, `address.country`, ... fields ?

Comment (kmonmousseau) [inline Tests/Provider/Nominatim/LocatorTest.php]: It seems that the returned `postcode` is the concaténation of all available postcodes of given `city`: for Montpellier, `"34000;34070;34080;34090"`.
I don't know if i should take the first one or not...

Comment (methylbro) [inline Tests/Provider/Nominatim/LocatorTest.php]: Don't bother yourself. Just pick the first one.

**Pull request merged**